File: Challenges/challenge6.js
// Make a function to remove duplicates from sorted array 

var removeDuplicates = function(nums) {
    var newNums = nums.filter((item, index) => {
        nums.indexOf(item) === index
    })
    newNums = newNums.slice(0, newNums.length);
    return newNums
};


var arr = [1,2,3,4,5,1,2,3,4,5]

console.log(removeDuplicates(arr)); 

File: Challenges/challenge6.py
# remove duplicates from the array 

def removeDuplicates(nums): 
    k = 1
    for i in range(1, len(nums)):
        if nums[i-1] != nums[i]:
            nums[k] = nums[i]
            k += 1
    return k
    


arr = [1,2,3,4,5,1,2,3,4,5]

print(removeDuplicates(arr))
